src/components/Acknowledgement/Acknowledgement.tsx

```diff
@@ -224,3 +233,69 @@ export const StreamingAcknowledgement = ({
     />
   )
 }
+
+export const ArbitrumBridgeAcknowledgement = (props: ArbitrumAcknowledgementProps) => {
+  const translate = useTranslate()
+  const [hasAgreed, setHasAgreed] = useState([false, false])
+
+  const isDisabled = useMemo(() => !hasAgreed.every(Boolean), [hasAgreed])
+
+  const handleAgree = useCallback(
+    (index: number) => (event: React.ChangeEvent<HTMLInputElement>) => {
+      const updatedAgreements = [...hasAgreed]
+      updatedAgreements[index] = event.target.checked
+      setHasAgreed(updatedAgreements)
+    },
+    [hasAgreed],
+  )
+
+  const checkboxTextColor = useColorModeValue('gray.800', 'gray.50')
+
+  const checkboxes = useMemo(
+    () => (
+      <Box py={4} textAlign='left' color={checkboxTextColor}>
+        <Checkbox onChange={handleAgree(0)} fontWeight='bold' py={2}>
+          {translate('bridge.arbitrum.waitCta')}
+        </Checkbox>
+        <Checkbox onChange={handleAgree(1)} fontWeight='bold' py={2}>
+          {translate('bridge.arbitrum.claimCta')}
+        </Checkbox>
+      </Box>
+    ),
+    [checkboxTextColor, handleAgree, translate],
+  )
+
+  const handleAcknowledge = useMemo(() => {
+    if (isDisabled) return
+
+    return props.onAcknowledge
+  }, [isDisabled, props])
+
+  const message = useMemo(
+    () => (
+      <>
+        <RawText>{translate('bridge.arbitrum.waitWarning')}</RawText>
```

Comment:
```suggestion
        <RawText as='span'>{translate('bridge.arbitrum.waitWarning')}</RawText>
```

or we will add another error:
<img width="555" alt="image" src="https://github.com/shapeshift/web/assets/14963751/51fc33b2-24df-4e51-999f-b94c12084865">

Reply:
2abb277125259cb89028218160dbe4e7e105cde5